Makefile

```diff
@@ -40,7 +40,7 @@ clean:
 build: generate
 	@echo "Creating GO binary"
 	mkdir -p target
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "$(GO_LDFLAGS)" -o target/pg-bifrost github.com/Nextdoor/pg-bifrost.git/main
+	CGO_ENABLED=0 GOOS=linux go build -ldflags "$(GO_LDFLAGS)" -o target/pg-bifrost github.com/Nextdoor/pg-bifrost.git/main
```

Comment:
this supports multi arch build?

Reply:
Yes but we don't produce multi arch docker images right now.